src/ui/image-preview/ImagePreview.tsx

```
@@ -0,0 +1,65 @@
+  };
+  const imagePreviewStyle = {
+    width: `${imageWidthRem}rem`,
+    height: `${imageHeightRem}rem`,
+  };
+
+  return (
+    <div
+      className={cn('relative overflow-hidden', handleClickImage ? 'cursor-pointer' : null, className)}
+      style={imagePreviewStyle}
+      onClick={handleClickImage}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)